api/lib/db/indexSync.js

```diff
@@ -5,6 +5,10 @@ let currentTimeout = null;
 
 // eslint-disable-next-line no-unused-vars
 async function indexSync(req, res, next) {
+  if (process.env.SEARCH && process.env.SEARCH.toLowerCase() !== 'true') {
```

Comment:
this seems redundant with line 12, try testing this and if it matches the expected change
```js
const searchEnabled = process.env?.SEARCH?.toLowerCase() === 'true';

if (!searchEnabled) {
  return;
}
```